### PR TITLE
AG-11910 - Allow arrow keys to move selected annotation drawing

### PR DESCRIPTION
**Pull request by manapeirov**
PR for https://ag-grid.atlassian.net/browse/AG-11910

### CODE REVIEW

File: packages/ag-charts-enterprise/src/features/annotations/annotations.ts [hunk 1]
```diff
@@ -1213,27 +1246,62 @@ export class Annotations extends _ModuleSupport.BaseModuleInstance implements _M
...
-        } else if (modifierKey && sourceEvent.key === 'x') {
-            this.state.transition('cut');
-            this.recordActionAfterNextUpdate('Cut annotation');
-        } else if (modifierKey && sourceEvent.key === 'v') {
-            this.state.transition('paste');
-            this.recordActionAfterNextUpdate('Paste annotation');
+        const translation = { x: 0, y: 0 };
+
+        const xStep = Math.max(context?.xAxis.scaleBandwidth() ?? 0, modifierKey ? 10 : 1);
+        const yStep = Math.max(context?.yAxis.scaleBandwidth() ?? 0, modifierKey ? 10 : 1);
```
Comment (lsjroberts) [inline packages/ag-charts-enterprise/src/features/annotations/annotations.ts]: This should be on ctrl or shift, not the meta key.

**Review by lsjroberts — APPROVED** [review]
LGTM, with one small issue. Good to merge once you're happy.

File: packages/ag-charts-enterprise/src/features/annotations/annotations.ts (hunk 1 above)
Comment (manapeirov) [reply packages/ag-charts-enterprise/src/features/annotations/annotations.ts]: good spot

**Review by alantreadway — APPROVED** [review]
LGTM

**Pull request merged**